generators/generator-transforms.js

```diff
@@ -52,7 +52,7 @@ const prettierTransform = function (defaultOptions) {
 const generatedAnnotationTransform = generator => {
     return through.obj(function (file, encoding, callback) {
         if (
-            !generator.configOptions.skipGeneratedFlag &&
+            generator.jhipsterConfig.withGeneratedFlag &&
```

Comment:
`!generator.configOptions.skipGeneratedFlag &&` condition is to let blueprints to skip the flag at runtime (post constructor) with `this.configOptions.skipGeneratedFlag = true`.

Since we are defaulting to disabled we can remove the condition.
Forcing true would need to queue the transform after constructor.

A alternative is to keep the condition and always queue the transform (remove condition from constructor).

Reply:
agree, let's remove this condition, you're right